### PR TITLE
VACMS-15055: Accelerated Publishing Install and enable computed_breadcrumbs.

### PR DESCRIPTION
**Pull request by schiavo**
## Description

Include breadcrumb data to JSON API to support Accelerated Publishing Next js implememtation. 
Relates to #14741 Spike: Breadcrumb modules for Drupal JSON:API 

## Testing done

Local testing comparing JSON API data with GraphQL data.

## Screenshots
Before
Screenshot-20230911141531-2338x2264

After
Screenshot-20230911141803-2056x2266

Comparison with GraphQL results
GraphQL
image

JSON API
image

## QA steps

1. Do this
   - [ ] Validate that breadcrumb data appears in JSON API e.g. /jsonapi/node/page
2. Then
   - [ ] Validate the UI is unaltered e.g. /node/add/page
3. Then validate Acceptance Criteria from issue
- [ ] Breadcrumb data is available in the JSON:API response for a node
- [ ] JSON:API breadcrumb data functionally matches GraphQL query data for equivalent queries/node types
- [ ] Installation of the module does not alter the editor experience in any way

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`
- [x] `Accelerated Publishing`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### CODE REVIEW

**Review by tjheffner — DISMISSED** [review]
This appears to be working for me.

I am not sure what the CI failure is about... there are only 49 lines changed in composer.lock here 😉 And they are changes we want... so 🤷 

```
Run thollander/actions-comment-pull-request@d61db783da9abefc3437960d0cce08552c7c004f
  with:
    comment_tag: check-composer-lock-changes
    message: The number of lines changed in composer.lock exceeds the acceptable threshold.
  
  - Lines changed: 50
  - Threshold: [2](https://github.com/department-of-veterans-affairs/va.gov-cms/actions/runs/6151340447/job/16691185057?pr=15193#step:5:2)00
  
  This is a warning only. Please review the changes and ensure that they are acceptable.
  
    GITHUB_TOKEN: ***
    mode: upsert
    create_if_not_exists: true
  env:
    LINES_CHANGED: [5](https://github.com/department-of-veterans-affairs/va.gov-cms/actions/runs/6151340447/job/16691185057?pr=15193#step:5:5)0
    THRESHOLD: 200
No comment has been found with asked pattern. Creating a new comment.
Error: Resource not accessible by integration
```

**Comment by ndouglas** [comment]
@tjheffner I believe it's a permissions issue:

```
Error: Resource not accessible by integration
```

`GITHUB_TOKEN` has read-only permissions on this PR because of the origin, so it can't write to the PR thread.

**Comment by ndouglas** [comment]
This one is more interesting:

CMS Facility editors are able to interact with Lovell -- Anonymous users can see the Lovell sidebar menu on Lovell TRICARE (failed)

**Comment by ndouglas** [comment]
interesting. On `main`:

Screenshot 2023-09-11 at 5 08 41 PM

On this preview:

Screenshot 2023-09-11 at 5 10 09 PM

So it looks like this test is checking every single link, and then this PR adds a new link that doesn't conform 🤔 

I'm not crazy about this test. I think it should restrict by a class or _something_. Because this is really an innocuous PR and it shouldn't break this test.

**Comment by ndouglas** [comment]
Opened an issue: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15196

**Comment by ndouglas** [comment]
@timcosgrove FYSA I think this PR is blocked by a bug in a Facilities test. Feel free to double-check me because it's late in the day for the ol' Nug Doug.

**Comment by ndouglas** [comment]
PR's merged, GHA shouldn't run on forks, so this should be unblocked now.

File: config/sync/core.entity_view_display.node.basic_landing_page.default.yml [hunk 1]
```diff
@@ -20,7 +20,6 @@ third_party_settings:
   field_group:
     group_metadata:
       children:
-        - field_meta_title
```
Comment (tjheffner) [inline config/sync/core.entity_view_display.node.basic_landing_page.default.yml:23]: are these non-breadcrumb changes intended?

**Review by tjheffner — COMMENTED** [review]
the JSON:API response looks to have the correct data now ✅ 
the node view page looks the same as it does without these modules ✅ 

I am curious about the number of non-breadcrumb related changes updated in the config. It appears more than a few node types have updates to things like meta tags, field_administration, image_loading, etc. Are those changes intentional?

**Review by tjheffner — APPROVED** [review]

**Comment by tjheffner** [comment]
config discrepancies resolved, jsonapi looks good and all the existing gql tests are passing. ✅

**Comment by ndouglas** [comment]
I'm seeing some differences in breadcrumbs on the node views:

[Tugboat](https://pr15193-g6gtrrtnfn89qfs9kqao6gtklygc7j6g.ci.cms.va.gov/outreach-and-events/events/61557)

Screenshot 2023-09-22 at 1 11 11 PM

[Prod](https://prod.cms.va.gov/outreach-and-events/events/61557)

Screenshot 2023-09-22 at 1 12 01 PM

**Comment by ndouglas** [comment]
@schiavo That seems reasonable to me, and I'm not aware of any functionality that requires the current behavior. No objection to merging this from me 👍🏻

**Comment by schiavo** [comment]
@ndouglas cool. IMHO the biggest concern is that this doesn't affect front end build. Which after some extensive poking around seems to be the case.

**Comment by ndouglas** [comment]
@swirtSJW This PR makes some (very minor) changes to some node views. Do you see an issue with this?

**Comment by swirtSJW** [comment]
From a facilities POV I don't see a problem.  There is parity with VAMC facilities on prod and with the FE
image

There is also parity with the new implementation on and FE and it actually improves it because Prod does not have parity.
image

VAMC Detail pages check out as no change to the CMS even though they never had parity on the FE
image


Same for Event list pages
image


However, the discrepancies @ndouglas noticed are on a product belonging to Public Websites, not facilities.  You will need to clear this with that team.   In general though, we are always trying to achieve parity with the FE behavior.

**Comment by jilladams** [comment]
@laflannery should take a look at this from an a11y perspective. 
Since it's CMS UX, @BlakeOrgan should also prob weigh in if he hasn't already.

**Comment by tjheffner** [comment]
@laflannery @BlakeOrgan  looking at CMS breadcrumbs only, and we'll have this PR environment working again for you ASAP.

The breadcrumbs will match prod CMS in most cases, but not all. Swirt's comment has a good breakdown of the differences between this PR, prod CMS, & the live site. Essentially some pages have gained a visible breadcrumb in the CMS that wasn't visible before. This new crumb matches what the breadcrumb visible on the live site in most cases. 

Essentially looking from an a11y+ux perspective that including an extra breadcrumb for the current page is kosher.

Testing on random nodes is fine.

**Comment by laflannery** [comment]
I actually like the new, additional breadcrumb, this makes everything much more expected and consistent. I have no issues with this new display

**Comment by BlakeOrgan** [comment]
Approved. I like the addition of displaying the current page within the breadcrumbs.

**Review by tjheffner — APPROVED** [review]

**Pull request merged**